server/bound_queries.cc

```diff
@@ -1449,6 +1449,13 @@ execute(RowProcessor processor,
     typedef std::map<RowKey, GroupMapValue> GroupByMapType;
     std::vector<GroupByMapType> accum(numBuckets);
 
+    for (const auto & c: select.clauses) {
+        if (c->isWildcard()) {
```

Comment:
You shouldn't use a function like isWildcard().  It makes the API too heavy.  You should compare the typeid() instead.